src/routes/_authed/campaigns/$dmUsername.$campaignSlug/notes/-components/editor/file-sidebar/sidebar-system-folder/character-system-folder/character-category-folder.tsx

```
@@ -0,0 +1,16 @@
+import { CategoryFolderButton } from "../generic-category-folder/category-folder-button";
+import { CharacterCategoryFolderContextMenu } from "./character-category-context-menu";
+import { CharacterNoteContextMenu } from "./character-note-context-menu";
+import { CHARACTER_CONFIG } from "~/components/forms/category-tag-dialogs/character-tag-dialog/types";
+
+interface CharacterCategoryFolderProps {}
+
+export const CharacterCategoryFolder = ({}: CharacterCategoryFolderProps) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)